libp2p/protocols/pubsub/floodsub.nim

```diff
@@ -85,9 +85,12 @@ method rpcHandler*(f: FloodSub,
 
         # forward the message to all peers interested in it
         var sent: seq[Future[void]]
+        # start the future but do not wait yet
         for p in toSendPeers:
           if p in f.peers and f.peers[p].id != peer.id:
             sent.add(f.peers[p].send(@[RPCMsg(messages: m.messages)]))
+        # wait for all the futures now
+        # TODO FIXME what to do with errors here??
```

Comment:
@dryajov Should we just leave `allFutures` and let it throw here? (repeating the question few times around)

Reply:
This one might actually not matter, we can't do much if we failed here either way. We can assume an optimistic approach here, if a message failed to be sent/forwarded to a peer that is unresponsive, the message won't be cached for that peer and it would either eventually fall off or resent, so being optimistic and ignoring errors for sends should be right thing to do overall.